sklearn/metrics/classification.py

```diff
@@ -221,8 +221,11 @@ def confusion_matrix(y_true, y_pred, labels=None, sample_weight=None):
 
     References
     ----------
-    .. [1] `Wikipedia entry for the Confusion matrix
+    .. [1] `Wikipedia entry for the Confusion matrix*
            <https://en.wikipedia.org/wiki/Confusion_matrix>`_
+                *NOTE: Different conventions exist for confusion matrix axes.
```

Comment:
This indentation doesn't work.

Reply:
I see the the problem. Sorry about that!